scripts/setup/ubuntu/install_deps.sh

```diff
@@ -14,7 +14,9 @@ DEPS=(
   g++
   gcc
   git
+  gnuplot # Litani (recommended) dependency
```

Comment:
Do we really need to add `gnuplot` as a dependency to RMC? It brings in a ton of other packages. Since the [Installation ](https://model-checking.github.io/rmc/install-guide.html) section of the tutorial refers to `install_deps.sh`, can we perhaps put `gnuplot` in a separate `install_dashboard_deps.sh` that is only run in CI?

Reply:
I liked this idea so I added it. The script is only run in the workflow when the dashboard is going to be generated.